backend/src/main/java/com/ambacam/transfert/passports/PassportReadTO.java

```diff
@@ -0,0 +1,82 @@
+package com.ambacam.transfert.passports;
+
+import java.util.Date;
+
+import com.ambacam.model.Autorite;
+
+public class PassportReadTO {
+
+    private Long id;
+    
+    private String numero;
+
+    private Date dateDelivrance;
+    
+    private Date dateExpiration;
+
+    private String lieuDelivrance;
+    
+    private Autorite delivrePar;
+
+    private String urlPhoto;
```

Comment:
Could you please remove this from the TO? We will create later a separated endpoint for uploading a photo.